src/hotspot/share/opto/machnode.hpp

```diff
@@ -539,6 +540,7 @@ class MachEpilogNode : public MachIdealNode {
   virtual uint size(PhaseRegAlloc *ra_) const;
   virtual int reloc() const;
   virtual const Pipeline *pipeline() const;
+  virtual uint size_of() const { return sizeof(MachEpilogNode); }
 
 private:
   bool _do_polling;
```

Comment:
Maybe move this field declaration up to the start of the class.

Reply:
Fixed